manifests/4.7/clusterresourceoverride-operator.v4.7.0.clusterserviceversion.yaml

```diff
@@ -298,7 +298,7 @@ spec:
                       - name: OPERAND_IMAGE
                         value: quay.io/openshift/clusterresourceoverride-rhel8:4.7
                       - name: OPERAND_VERSION
-                        value: 1.0.0
+                        value: 4.7.0
```

Comment:
if this needs to match `version:` below exactly, you should be aware that will be updated per `art.yaml` to look like e.g. `4.7.0-202012071234.p0`; it would need an addition in `art.yaml` for this line to be similarly updated.